rllib/BUILD

```diff
@@ -1240,7 +1240,7 @@ py_test(
 py_test(
     name = "test_preprocessors",
     tags = ["team:ml", "models"],
-    size = "medium",
+    size = "large",
```

Comment:
Is this growth expected (e.g. new tests were added)? Or is there a regression in the functionality causing it to slow down?

Reply:
I think we had some runtime slowdown regressions in some of our TF2 algorithms, which could have lead to the growth in runtime of the test. I'm increasing to large for now, and will monitor closely. We could fix the seed for this experiment. I'll follow up offline to see if this is ok.

Reply:
+1

Reply:
After fixing the seed, do we still need to increase to large?

Reply:
I'll get back to you after letting the CI run -- if the CI throws a warning that the amount of time needed is unnecessary, which it does if the test marker is too big compared to the runtime of the test, then I'll reduce the size, otherwise I'll leave as is.

Reply:
the large test size ended up being appropriate.